### PR TITLE
New updates for `/updates`

### PR DESCRIPTION
**Pull request by bladey**

### CODE REVIEW

File: docs/pages/updates/index.tsx [hunk 1]
```diff
@@ -167,7 +171,75 @@ export default function WhatsNew() {
           gap: 0,
         })}
       >
-        <Timeline date="6th September 2021" isLatest />
+        <Timeline date="21st September 2021" isLatest />
+        <Box heading="Keystone Community Q&A + YouTube launch ">
+          Taking the chance to introduce the core team and address some community questions in
+          person, the team held an online Q&A event and launched a Youtube channel at the same time.{' '}
```
Comment (timleslie) [inline docs/pages/updates/index.tsx:177]:
```suggestion
          person, the team held an online Q&A event and launched a YouTube channel at the same time.{' '}
```

File: docs/pages/updates/index.tsx [hunk 2]
```diff
@@ -167,7 +171,75 @@ export default function WhatsNew() {
...
+          Our CLI app now uses <InlineCode>SQLite</InlineCode> under the hood so you don’t have to
+          spend time on DB config when trying out new ideas. We also updated the 
+          <Link href="/docs/walkthroughs/getting-started-with-create-keystone-app">
+            <a>getting started walkthrough</a>
+          </Link>{' '}
+          to reflect this improvement.
+        </Box>
+        <Timeline date="15th September 2021" />
+        <Box heading="Deployment examples for Heroku and Railway">
+          Learn how to get your Keystone project on the web with out new one-click starters for{' '}
```
Comment (timleslie) [inline docs/pages/updates/index.tsx:198]:
```suggestion
          Learn how to get your Keystone project on the web with our new one-click starters for{' '}
```

File: docs/pages/updates/index.tsx [hunk 3]
```diff
@@ -167,7 +171,75 @@ export default function WhatsNew() {
...
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Railway
+          </a>
+          .
+        </Box>
+        <Timeline date="9th September 2021" />
+        <Box heading="Prisma Korea Meetup">
+          Jed spoke at the Prisma Korea Meetup, covering V6 general availability, User-facing
```
Comment (timleslie) [inline docs/pages/updates/index.tsx:218]: The capitalisation here seems a bit haphazard here, not quite sure what it should be though 🤔

Comment (bladey) [reply docs/pages/updates/index.tsx]: Totally, adjusted.

**Review by raveling — APPROVED** [review]

**Pull request merged**